src/Serializer/AbstractItemNormalizer.php

```diff
@@ -297,6 +297,7 @@ private function denormalizeRelation(string $attributeName, PropertyMetadata $pr
             ($propertyMetadata->isWritableLink() && \is_array($value))
         ) {
             $context['resource_class'] = $className;
+            $context['api_resource_is_relation'] = true;
```

Comment:
Wouldn't be easier to just set `api_allow_update` to true for the subcontext in this case instead of introducing a new flag?

Reply:
Apparently it works, and probably fixes the `src/Serializer/ItemNormalizer.php` that triggers the same error that I missed, I will do this.